src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -1008,9 +1018,10 @@ public HostedPaymentPageFormDescriptor buildFormDescriptor(final HostedPaymentPa
         return httpClient.doPost(uri, fields, queryParams, HostedPaymentPageFormDescriptor.class);
     }
 
-    public HostedPaymentPageFormDescriptor buildFormDescriptor(final ComboHostedPaymentPage comboHostedPaymentPage, final Map<String, String> pluginProperties, final String createdBy, final String reason, final String comment) throws KillBillClientException {
+    public HostedPaymentPageFormDescriptor buildFormDescriptor(final ComboHostedPaymentPage comboHostedPaymentPage, List<String> controlPluginNames, final Map<String, String> pluginProperties, final String createdBy, final String reason, final String comment) throws KillBillClientException {
```

Comment:
This will break the Kill Bill server tests. Can you add a second (backward compatible) signature, like above?